polkadot-parachain/src/chain_spec/default.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Default parachain ChainSpec based on testing rococo parachain runtime
+
+use crate::chain_spec::{get_from_seed, Extensions, SAFE_XCM_VERSION};
+use cumulus_primitives_core::ParaId;
+use hex_literal::hex;
+use parachains_common::AccountId;
+use polkadot_service::chain_spec::get_account_id_from_seed;
+use rococo_parachain_runtime::AuraId;
+use sc_chain_spec::ChainType;
+use sp_core::{crypto::UncheckedInto, sr25519};
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type DefaultChainSpec =
+	sc_service::GenericChainSpec<rococo_parachain_runtime::GenesisConfig, Extensions>;
+
+pub fn get_chain_spec() -> DefaultChainSpec {
```

Comment:
would be great to have some doc.
when you see it first time, its really hard to say what is meant for

Reply:
well, to the other comments,
I did not touch any code (it stays like it was before), I just move it to `default.rs`,
main reason why I did this cleanup, was because lots of stuff depended on stuff from `rococo_parachain_runtime` (ChainSpec, AuraId, Genesis...)
```
pub type ChainSpec =
	sc_service::GenericChainSpec<rococo_parachain_runtime::GenesisConfig, Extensions>;
```

but thank you for your comments, now I can clean it little bit more :)

Reply:
as I said before, now everything is align to reflect parachains/runtimes structure, so all this Default/Generic stuff is changed to RococoParachain